data/statuses.js

```diff
@@ -718,6 +718,14 @@ let BattleStatuses = {
 		num: 0,
 		duration: 3,
 		onStart(pokemon) {
+			if (pokemon.species === 'Eternatus-Eternamax') { // Special for Eternatus' Eternamax forme
+				this.add('-start', pokemon, 'Eternamax');
```

Comment:
Do we need a special start message for emax?

Reply:
In game it just shows "Eternatus appeared" iirc, ill remove this

Reply:
I'm adding one just so it can differentiate